hathor/simulator/fake_connection.py

```diff
@@ -80,6 +80,35 @@ def disable_idle_timeout(self):
         self._proto1.disable_idle_timeout()
         self._proto2.disable_idle_timeout()
 
+    def is_synced_or_error(self) -> bool:
+        """Short-hand check that can be used to make "step loops" without having to guess the number of iterations."""
+        from hathor.p2p.states.ready import ReadyState
+        conn1_aborting = self._proto1.aborting
+        conn2_aborting = self._proto2.aborting
+        if conn1_aborting or conn2_aborting:
```

Comment:
Is aborting considered an error?

Reply:
I think `is_synced_or_error` isn't a good name. What do you think of renaming it to `can_step` (with flipped booleans)? It returns `True` when either connection aborts because it doesn't make sense to step aborted connections.

Reply:
Renamed.

Reply:
Seems good. Thanks!